provisioning/transport/http/src/ProvisioningTransportHandlerHttp.cs

```diff
@@ -148,10 +148,34 @@ public ProvisioningTransportHandlerHttp()
 
                     cancellationToken.ThrowIfCancellationRequested();
 
-                    operation = await client.RuntimeRegistration.OperationStatusLookupAsync(
+                    try
+                    {
+                        operation = await client.RuntimeRegistration.OperationStatusLookupAsync(
```

Comment:
Q - does `RegisterAsync()` throw 429, or only `OperationStatusLookupAsync()`? This wouldn't work if `RegisterAsync()` were to throw a throttling exception.

Reply:
OperationStatusLookupAsync returns RegistrationOperationStatus so it should be OperationStatusLookupAsync which throws 429